backend/src/main/entity/User.ts

```diff
@@ -67,6 +67,7 @@ export class User {
   // todo: we need something more secure
   @Column({
     unique: true,
+    select: false, // https://typeorm.io/#/select-query-builder/hidden-columns
```

Comment:
Anytime we access a user, we don't want to get their password. If in any case we need a user's password, we need to use a query builder to specifically ask for the password